insights/tests/client/apps/test_compliance.py

```diff
@@ -304,6 +304,35 @@ def test_find_scap_policy(config, call):
     compliance_client.profile_files = lambda: ['/something']
     assert compliance_client.find_scap_policy('ref_id') == PATH
 
+@patch("insights.client.config.InsightsConfig")
+def test_find_scap_policy_with_one_datastream_file(config):
+    compliance_client = ComplianceClient(config)
+    dir1 = "/usr/share/xml/scap/ssg/content/ssg-rhel9-ds.xml"
```

Comment:
I'm not sure if it's a good idea to write to these paths in the testing environment. Can we rather mock the path somewhere in the `find_scap_policy` to something in the `tmp`? Or even better to create fixtures and mock the path there?

Reply:
Yeah, I wasn't sure about this either but I didn't know how else to test this line `filenames = findall('/usr/share/xml/scap/.+xml', grep)` in `find_scap_policy` without using the correct path or mocking `findall`. 
But when you say "mock the path" do you mean use a `tmp` path instead of the `/usr/share/xml/scap/ssg/content/` path? I could mock the `findall` method here `filenames = findall('/usr/share/xml/scap/.+xml', grep)` to use the a tmp path instead of the `/usr/share/xml/scap/` path.

Reply:
Extract the path into a constant (or a variable?) and try to override it from the test framework.